tasklist/importer-850/src/main/java/io/camunda/tasklist/zeebeimport/v850/record/RecordImpl.java

```diff
@@ -48,6 +48,8 @@ public class RecordImpl<T extends RecordValue> implements Record<T> {
 
   private Map<String, Object> authorizations;
 
+  private long requestId;
```

Comment:
❓Looks fine to me but do we need someone from Tasklist team to review this part?

Reply:
@houssain-barouni would this be something you might review?

Reply:
Thanks @marcosgvieira ! 🙇‍♂️